### ISSUE TITLE
Defining schema from json fails if type of variable is a list

### ISSUE THREAD

**Issue by SebastienBocquet** [terminusdb/terminusdb-client-python#396]
**Describe the bug**

An error arises when using from_json_schema() method, in the case where the type of a variable is list.
The call to convert_property method fails because it expects two mandatory arguments, and only one is passed
at line 797 of schema/schema.py.

**To Reproduce**

```
client = Client("http://127.0.0.1:6363/")
client.connect(db="gemseo")
data_schema = WOQLSchema()
data_class_name = "StraightBeamModelOutput"
output_grammar_file = "StraightBeamModel_output.json"
json = JSONGrammar(data_class_name, schema_file=output_grammar_file)
data_schema.from_json_schema(data_class_name, json_schema=json.schema_dict)
```

with the json file:

[StraightBeamModel_output.json.txt](https://github.com/terminusdb/terminusdb-client-python/files/11428294/StraightBeamModel_output.json.txt)

**Expected behavior**

Schema is correctly added.

**Error logs**

```
/home/sebastien.bocquet/PycharmProjects/vims/.tox/py39/bin/python /home/sebastien.bocquet/PycharmProjects/vims/src/vims/database_facilities/add_schemas.py 
Traceback (most recent call last):
  File "/home/sebastien.bocquet/PycharmProjects/vims/src/vims/database_facilities/add_schemas.py", line 25, in <module>
    add_model_output_schema(client, "StraightBeamModel")
  File "/home/sebastien.bocquet/PycharmProjects/vims/src/vims/database_facilities/add_schemas.py", line 21, in add_model_output_schema
    data_schema.from_json_schema(data_class_name, json_schema=json.schema_dict)
  File "/home/sebastien.bocquet/PycharmProjects/vims/.tox/py39/lib/python3.9/site-packages/terminusdb_client/schema/schema.py", line 819, in from_json_schema
    class_dict[prop_name] = convert_property(prop_name, prop)
  File "/home/sebastien.bocquet/PycharmProjects/vims/.tox/py39/lib/python3.9/site-packages/terminusdb_client/schema/schema.py", line 797, in convert_property
    prop_type = convert_property(prop["items"])
TypeError: convert_property() missing 1 required positional argument: 'prop'
```

**System information (please complete the following information):**
 - OS: Ubuntu 16.04
 -  terminus-client-python version: 10.2.2

**Comment by SebastienBocquet** [comment]
Hi,
Could you have a look at this problem?
Thanks

**Comment by GavinMendelGleason** [comment]
Should be fixed in #409 (when merged)
